epam-promo/components/tables/DataTable.scss

```diff
@@ -47,4 +47,14 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
+
+    &-icon {
+        align-items: center;
+    }
+
+    &-title {
+        font-weight: 600;
```

Comment:
for bold text we usually use separate font-family: $sans-semibold, which already has bold style by default